File: java-main/java-main/NestedforloopAssi/p28.java
/*
10
I H
7 6 5
D C B A*/

class Demo{

	public static void main(String[]args){

		int num=10;
		int N=4;
		char ch='J';

		for(int i=1;i<=N;i++){
			for(int j=1;j<=i;j++){
				if(i%2!=0){
					System.out.print(num + " ");
				}else{
					System.out.print(ch + " ");
				}
				num--;
				ch--;
			}
			System.out.println();
			
		}
	}
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/CPP/codes/p28.cpp
#include<iostream>
class Demo{
	int x=10;
	int y=20;
	public:
	Demo(){
		std::cout<< "In Contructor"<<std::endl;
	}
	void fun(){
		std::cout<< x <<std::endl;
		std::cout<< y <<std::endl;
	}
};
int main(){
	Demo obj;
	obj.fun();
	return 0;
}

